### DISCUSSION TITLE
What am I missing?

### DISCUSSION THREAD

**Discussion by aidinzolghadr** [hsoft/dupeguru#1125] [Q&A]
Hi,

Isn't dupeGuru supposed to check the content when in standard mode? I was assuming using hash algorithms to determine if files are actually duplicates but it doesn't seem to be the case so I ran a test which confused me even more.

I made a new folder with three text files inside it which they all have a '1' character in them but with different file names:

1. "1.txt" 
2. "2.teext" 
3. "3.txt"

Point is second one has the content of the first one but with different name and third one just a normal extension but sharing the same content.

But the app doesn't find any duplicate in it.

I've tried to read the docs but can't find what I'm missing. My best guess was a "Preferences" menu which I can't find. Using 4.3.1 on Win 10.

Thanks

**Comment by arsenetar** [comment]
By default the preferences has a value for ignoring small files, you likely want to modify that value in this case.  You can get to that by clicking the more options on the main dialog.

**Reply by aidinzolghadr** [reply]
Thanks, go tit. Is it doing some kind of checksum/hashing underneath to be sure two non-text files are identical? And how I can change it?
